update-common/manifests/fake0.toml

```diff
@@ -0,0 +1,87 @@
+# This is an artifact manifest that generates fake entries for all components.
```

Comment:
How about calling this `fake-old.toml` and adding a comment here that the only difference from `fake.toml` is the version numbers of everything?

Reply:
#7832 added a (non-semver-containing) 2.0.0 and made the test use that, so e019fda93 just tries to revert back to 1.0.0 and removes this file entirely.